docs/maatregelen/formuleren_doelstellling.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Formuleren doelstelling
+toelichting: Het doel en de eventuele subdoelen van het algoritme moeten zo specifiek mogelijk zijn geformuleerd, en waar mogelijk gekwantificeerd. 
+vereiste:
+- zorgvuldigheidsbeginsel
+levenscyclus:
+- probleemanalyse
+bouwblok:
+- governance
+rollen:
+- projectleider
+- opdrachtgever
+- domeinspecialist
+hide:
+- navigation
+---
+
+<!-- tags -->
+
+## Maatregel
+
+Het doel en de eventuele subdoelen van het algoritme moeten zo specifiek mogelijk zijn geformuleerd, en waar mogelijk gekwantificeerd. 
+Maak de consequenties van het algoritme specifiek en leg deze op het juiste niveau vast.
```

Comment:
op het juiste niveau vastleggen, betekent dat dat bepaalde _mensen_ op niveau ervan af moeten weten, of dat het in een bepaald _format_ moet worden vastgelegd? Denk dat deze zin nu nog net iets te verwarrend is.